.github/workflows/package.yml

```
@@ -26,18 +26,18 @@ jobs:
     strategy:
       matrix:
         include:
+          - image: ubuntu2404
+            config-zstd: OFF
```

@seladb you should also set it here

```suggestion
            config-zstd: OFF
            additional-pip-flags: --break-system-packages
```